### PR TITLE
Fixing warnings in 3party lib.

### PR DESCRIPTION
**Pull request by bykoianko**
@ygorshenin @mpimenov @greshilov  PTAL

### CODE REVIEW

**Comment by bykoianko** [comment]
@ygorshenin PTAL

File: CMakeLists.txt [hunk 1]
```diff
@@ -368,6 +362,16 @@ add_subdirectory(3party/sdf_image)
 add_subdirectory(3party/stb_image)
 add_subdirectory(3party/succinct)
 
+if (PLATFORM_DESKTOP)
+  add_subdirectory(3party/libtess2)
+endif()
+
+add_compile_options(
```
Comment (ygorshenin) [inline CMakeLists.txt]: Please, add a comment that only options related to warnings can be placed here, between 3party and our code. Other options must be set before all ```add_subdirectory``` call.

Comment (bykoianko) [reply CMakeLists.txt]: Done

**Comment by bykoianko** [comment]
@ygorshenin PTAL

**Pull request merged**